web/ui/react-app/src/pages/Status.tsx

```diff
@@ -116,7 +104,9 @@ const Status: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix = '' })
         );
       })}
     </>
-  ) : null;
+  );
 };
 
-export default Status;
+export default ({ pathPrefix = '' }: RouteComponentProps & PathPrefixProps) => (
+  <Fetch urls={endpoints.map(e => `${pathPrefix}${e}`)}>{statusRenderer}</Fetch>
```

Comment:
Nice.

Reply:
I'm accepting this as 👍 to refactor other components as well :)